src/pharmpy/tools/resmod/tool.py

```diff
@@ -35,26 +35,31 @@ def create_workflow(model=None, groups=4):
     task_base_model = Task('create_base_model', _create_base_model)
     wf.add_task(task_base_model, predecessors=start_task)
 
-    task_iiv = Task('create_iiv_on_ruv_model', _create_iiv_on_ruv_model)
-    wf.add_task(task_iiv, predecessors=task_base_model)
-    task_power = Task('create_power_model', _create_power_model)
-    wf.add_task(task_power, predecessors=task_base_model)
-    task_combined = Task('create_combined_error_model', _create_combined_model)
-    wf.add_task(task_combined, predecessors=task_base_model)
-
     tasks = []
-    for i in range(1, groups):
-        tvar = partial(_create_time_varying_model, groups=groups, i=i)
-        task = Task(f"create_time_varying_model{i}", tvar)
-        tasks.append(task)
-        wf.add_task(task, predecessors=task_base_model)
-
-    fit_wf = create_fit_workflow(n=3 + groups)
-    wf.insert_workflow(
-        fit_wf, predecessors=[task_base_model, task_iiv, task_power, task_combined] + tasks
-    )
-
-    task_post_process = Task('post_process', post_process)
+    if 'IIV_on_RUV' not in skip:
+        task_iiv = Task('create_iiv_on_ruv_model', _create_iiv_on_ruv_model)
+        tasks.append(task_iiv)
+        wf.add_task(task_iiv, predecessors=task_base_model)
+
+    if 'power' and 'combined' not in skip:
```

Comment:
This is not what you intend. It will be testing `'combined' not in skip` only.

Reply:
Thank you for your comment! I forgot it...I correct the code now!

Reply:
It should be `if 'power' not in skip and 'combined' not in skip:`